### ISSUE TITLE
 The accuracy of the validation set remains unchanged during the training process.

### ISSUE THREAD

**Issue by ttxs001** [BarryAlllen/PIMoG-Improved#1]
There is a problem during the training process of the provided pimog code (using the COCO data set): No matter how the ratio of the training set to the verification set is modified, the accuracy of the verification set remains at about 50%, and as the number of training rounds increases, the verification The set accuracy will not change much either. Have you encountered this kind of problem during the test? If so, how did you solve it?
![屏幕截图 2024-01-29 191938](https://github.com/BarryAlllen/PIMoG-Improved/assets/101035395/80e97540-9bf9-4254-a96c-2a6dba93d438)

**Comment by BarryAlllen** [comment]
> There is a problem during the training process of the provided pimog code (using the COCO data set): No matter how the ratio of the training set to the verification set is modified, the accuracy of the verification set remains at about 50%, and as the number of training rounds increases, the verification The set accuracy will not change much either. Have you encountered this kind of problem during the test? If so, how did you solve it? 

There is a probability that his learning effect may not be good, resulting in a fluctuation of about 50%. In this case, you can stop and restart it

**Comment by ttxs001** [comment]
May I ask why the accuracy rate of the train_mask_ScreenShooting_2023-12-26-15-35.txt file in logsbackup/backup is so high in the first round? What data set did you use and how did you train it? Do you want to configure initial parameters manually?

**Comment by BarryAlllen** [comment]
> May I ask why the accuracy rate of the train_mask_ScreenShooting_2023-12-26-15-35.txt file in logsbackup/backup is so high in the first round? What data set did you use and how did you train it? Do you want to configure initial parameters manually?

Just use the parameters I set up

**Comment by ttxs001** [comment]
Thank you very much. I saw that you made a model with input images of 128*128 and 256*256. Can you increase the resolution to 512 or even 1k to make the watermarked image clearer?In addition, I noticed that the original paper used 2080Ti for training. I used Tesla T4 for training of 128 resolution images. 100 rounds took about 10 days. Have you encountered the problem of too long training time? Are some accelerated training methods used?
